lib/sunrise/sunrise.php

```diff
@@ -58,7 +60,7 @@ function handle_not_found_error( $error_type ) {
 	if ( $is_healthcheck ) {
 		http_response_code( 200 );
 		header( 'Content-type: text/plain' );
-		echo sprintf( '%s not found; but still OK', htmlspecialchars( $error_type ) );
+		printf( '%s not found; but still OK', esc_html( $error_type ) );
```

Comment:
> All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'htmlspecialchars'.

`esc_html()` is defined in [`formatting.php`](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-includes/formatting.php#L4498). That file is `require`'d [before](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-settings.php#L109) the system [loads](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-includes/ms-settings.php#L47) `sunrise.php` from [`ms-settings.php`](https://github.com/WordPress/WordPress/blob/03bbf916a96869629a1978a448e8a52a32cef5fa/wp-settings.php#L141)